src/route-handlers/describe-workflow/describe-workflow.ts

```diff
@@ -23,21 +28,119 @@ export default async function describeWorkflow(
         runId: decodedParams.runId,
       },
     });
+    const res: DescribeUnArchivedWorkflowResponse = merge(
```

Comment:
nit: you can add spaces between different logical blocks of code to make it a bit easier to read. It's just a preference thing at the end of the day but I think it's nice to do.